lib/producer.ts

```diff
@@ -221,19 +221,24 @@ function nativePrebuildInstall(target: Target, nodeFile: string) {
     fs.copyFileSync(nodeFile, `${nodeFile}.bak`);
   }
 
+  const napiVersions = JSON.parse(
+    fs.readFileSync(path.join(dir, 'package.json'), { encoding: 'utf-8' })
+  )?.binary?.napi_versions;
+
+  const options = [
+    '--platform',
+    platform[target.platform],
+    '--arch',
+    target.arch,
+  ];
+
+  if (napiVersions == null) {
```

Comment:
Shouldn't you change `--runtime` option based on this? https://github.com/prebuild/prebuild-install#help

Reply:
See #1695. prebuild-install already does that.